extensions/multiple-references-extension/src/index.tsx

```diff
@@ -16,6 +17,7 @@ init<FieldExtensionSDK>((sdk) => {
         viewType="link"
         sdk={fieldSdk}
         isInitiallyDisabled={true}
+        hasCardEditActions={true}
```

Comment:
Why was this added?

Reply:
This was a mandatory prop in the component :/